ldoc/html/ldoc_ltp.lua

```diff
@@ -295,7 +300,7 @@ return [==[
 <table class="module_list">
 # for m in mods() do
 	<tr>
-		<td class="name"  $(nowrap)><a href="$(no_spaces(kind))/$(m.name).html">$(m.name)</a></td>
+		<td class="name $(nowrap)"><a href="$(no_spaces(kind))/$(m.name).html">$(m.name)</a></td>
```

Comment:
Is this really a CSS class or an HTML property string?

Reply:
This is what I was referring to in #360. But according to [html.com](https://html.com/attributes/td-nowrap/), it *is* an attribute for `td` tag.

Reply:
In that case, this bit of patch is *wrong* and should be dropped.

@jeblad are you still around?

Reply:
nowrap is not part of html 5.